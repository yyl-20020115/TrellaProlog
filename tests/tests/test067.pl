main :-
	( call(writeq, 'OK here') ->
		(nl, writeq('OK no error'), nl) ; (nl, writeq('OOPS was error'), nl)
	),
	writeq('OK done (3rd line)'), nl.

:- initialization(main).
