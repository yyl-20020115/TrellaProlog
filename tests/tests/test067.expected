'OK here'
'OK no error'
'OK done (3rd line)'
